game/sprite-backend.rpy

```diff
@@ -1082,9 +1086,57 @@ image natsuki idle = ConditionSwitch(
     predict_all = True
 )
 
+image natsuki test = ConditionSwitch(
+    "Natsuki.getMouseIsLeft() and Natsuki.getMouseIsAbove()", "natsuki 5utlsml",
+    "Natsuki.getMouseIsRight() and Natsuki.getMouseIsAbove()", "natsuki 5utrsml",
+    "Natsuki.getMouseIsLeft() and Natsuki.getMouseIsBelow()", "natsuki 5udlsml",
+    "Natsuki.getMouseIsRight() and Natsuki.getMouseIsBelow()", "natsuki 5udrsml",
+    "Natsuki.getMouseIsLeft()", "natsuki 5ullsml",
+    "Natsuki.getMouseIsRight()", "natsuki 5ulrsml",
+    "Natsuki.getMouseIsAbove()", "natsuki 5uupsml",
+    "Natsuki.getMouseIsBelow()", "natsuki 5udwsml",
+    "True", "natsuki 5unmsml",
+    predict_all = True
+)
+
+label dynatest:
+    show natsuki test
+    n ""
+    return
+
 # Idle images for ENAMORED+
 image natsuki idle enamored:
     block:
+        choice:
+            ConditionSwitch(
+                "Natsuki.getMouseIsLeft() and Natsuki.getMouseIsAbove()", "natsuki 5utlsml",
+                "Natsuki.getMouseIsRight() and Natsuki.getMouseIsAbove()", "natsuki 5utrsml",
+                "Natsuki.getMouseIsLeft() and Natsuki.getMouseIsBelow()", "natsuki 5udlsml",
+                "Natsuki.getMouseIsRight() and Natsuki.getMouseIsBelow()", "natsuki 5udrsml",
+                "Natsuki.getMouseIsLeft()", "natsuki 5ullsml",
+                "Natsuki.getMouseIsRight()", "natsuki 5ulrsml",
+                "Natsuki.getMouseIsAbove()", "natsuki 5uupsml",
+                "Natsuki.getMouseIsBelow()", "natsuki 5udwsml",
+                "True", "natsuki 5unmsml",
+                predict_all = True
+            )
+            pause 4
+            "natsuki 3fchbll"
+            pause 1
+            "natsuki 3fchsml"
+            pause 3
```

Comment:
I really love this feature and it motivates a lot to interact with Natsuki more! It also add another layer of dimension as this is one of the rarely times the player has the ability to interact with her nonverbal! 

We should explain in the wiki how to trigger it or what affinity level is needed, as I can see many user will ask it.

After almost 10min of waiting the eye idles triggered only 3 times for me, I think it is fine as it is random, but I feel like the pause til the the idle ended should be a bit longer, so the player can play around with the new feature a bit more. Perhaps it could also be longer the higher the affinity tier is actually!